pytorch_lightning/plugins/training_type/horovod.py

```diff
@@ -158,3 +155,8 @@ def gather_all_tensors(self, result: Union[torch.Tensor], group: Optional[Any] =
         gathered = hvd.allgather(result)
         gathered_result = list(gathered.split(1, dim=0))
         return gathered_result
+
+    def post_backward(self, closure_loss: torch.Tensor, should_accumulate: bool, optimizer: Optimizer, opt_idx: int):
```

Comment:
shall we unify `opt_idx` and `optimizer_idx` across codebase?

Reply:
Let's do that in another PR.